.pytool/Plugin/LineEndingCheck/Readme.md

```diff
@@ -0,0 +1,19 @@
+# Line Ending Check Plugin

```

Comment:
Should there also be a comment that this plugin is really only relevant if the workspace is checked out on Linux or with autocrlf=false

Reply:
Added guidance in https://github.com/microsoft/mu_basecore/pull/123/commits/d69740acdb0747e1e37dd1c867a271212e16b73c